docker-compose.yaml

```diff
@@ -31,8 +31,7 @@ x-default-fabric-env: &x-default-fabric-env
 
 services:
   mc:
-    build:
-      dockerfile: Dockerfile
+    image: trihatmaja/tomcrusade-minecraft-server:latest
```

Comment:
For extensibility, what if the image name should be `trihatmaja/tomcrusade-mc-server-fabric-vanilla-1.21:latest`?

Reply:
Also, i think we should rename Dockerfile to match with minecraft version, like `Dockerfile.fabric-vanilla.1-21`, so that we can add additional Dockerfile image configuration for future minecraft versions and still retain previous ones. 

If so, also add `file: ./Dockerfile.fabric-vanilla.1-21` to the github actions file